src/lib/components/TopNavBar.svelte

```diff
@@ -37,7 +39,13 @@
         if (tab === 'bible') {
             showBookChapterVerseMenu = true;
         } else if (tab === 'guide' || tab === 'resources') {
-            showBookPassageSelectorPane = true;
+            const currentGuideId = $currentGuide?.id;
+            if (currentGuideId && PredeterminedPassageGuides.includes(currentGuideId)) {
+                showBookPassageSelectorPane = true;
+            }
+            {
+                showBookChapterVerseMenu = true;
+            }
```

Comment:
Are the braces on line 46 and 48 needed?

Reply:
Good catch it's supposed to be an else block but somehow I forgot the else part